coffee-quarkus-extensions/coffee-module-etcd-extension-parent/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>hu.icellmobilsoft.coffee</groupId>
+        <artifactId>coffee-quarkus-extensions</artifactId>
+        <version>2.11.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>coffee-module-etcd-extension-parent</artifactId>
+
+    <packaging>pom</packaging>
+
+    <name>Coff:ee module etcd Extension - Parent</name>
+    <description>Quarkus Extensions - Coff:ee module etcd extension</description>
+
+    <modules>
+        <module>coffee-module-etcd-extension</module>
+        <module>coffee-module-etcd-extension-deployment</module>
+    </modules>
+
+    <build>
+        <pluginManagement>
```

Comment:
ide miért kell pluginManagement? Ezt nem egy fentebbi szint már megoldja?

Reply:
ezt szerintem a quarkus plugin create-extension-je pakolja bele alapból. 
A quarkus.version property meg a nem quarkus pluginok beállítása miatt nem lenne jó helyen a coffee root pom-jába. viszont coffee-quarkus-extensions pomjába átkerülhetnének, és akkor már nem lenne extensionönként újra és újra definiálva

Reply:
nem néztem, azt hittem oda annó beletettem amikor csináltam, de igen erre gondolnék :D